packages/eslint-config/package.json

```diff
@@ -15,6 +15,14 @@
     "airbnb",
     "prettier"
   ],
+  "files": [
+    "cypress.js",
+    "index.js",
+    "jest.js",
+    "react.js",
+    "typescript.js",
+    "vue.js"
```

Comment:
@illepic did you say to remove all the vue stuff?

Reply:
Yep let's remove vue.js

Reply:
Support for vue.js can be manually added later.